packages/kit/src/core/dev/index.js

```diff
@@ -317,7 +326,8 @@ class Watcher extends EventEmitter {
 					res.end(e.stack);
 				}
 			});
-		});
+
+		await set_listener(this.server, this.port, this.host);
```

Comment:
`set_listener` is a weird name for this since the listener isn't passed in. can we just call it `listen` instead?